### ISSUE TITLE
New Optional Support for Nurse Burn

### ISSUE THREAD

**Issue by Fackonami** [BullyWiiPlaza/JMaster-Duel-Bot#27]
I was wondering if it was possible for the Nurse Burn bot to have support for certain (optional) cards for customization purposes, since it already has support for Ring of Destruction and other cards from Dynamite and Self Burn.

Suggestions:
- Maxx C (incase opponent goes first. activate during opponent’s draw phase, to discourage them from spamming)
- Triple Tactics Talent (incase they use something like Ash, always select draw 2)
- Triple Tactics Thrust (not in the game yet but when it is, the bot could support it, search Spell if they have a monster, set Trap if they don’t)

These are just suggestions.

**Comment by BullyWiiPlaza** [comment]
Yes, it would be possible when it's implemented. It might be implemented in the future but it's not very important right now. There are probably enough alternative supported cards.

**Comment by Fackonami** [comment]
Yes there actually are other options:

- Dark Cure
- Secret Barrel
- Just Desserts
- Chain Strike
- Secret Blast

**Comment by Fackonami** [comment]
I was wondering if you could add support for Final Countdown (there are turn passing bots in Festivals, and I just beat one of them with it.) It’s a very simple addition, and I’m not asking for a FC Stall Deck. Maybe there could even be an option for it?

**Comment by BullyWiiPlaza** [comment]
Hi @Fackonami, unfortunately, the card is pretty bad and you're better off running a lot of other things. Waiting 20 turns for a duel, even if you win, is pretty wasteful in terms of time when using the bot. Duels should be faster. Sure, I could add support for it since it's a "simple card", but I have many other things to do, it would be smarter to prioritize differently.
